indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1950,6 +1950,45 @@ public Boolean apply(SeekableStreamIndexTaskRunner.Status status)
     // make sure the checkpoints are consistent with each other and with the metadata store
 
     verifyAndMergeCheckpoints(taskGroupsToVerify.values());
+
+    // A pause from the previous Overlord's supervisor, immediately before leader change
+    // can lead to tasks being in a state where they are active but do not read.
+    // If this is the first run, resume all existing active tasks to be safe
+    if (getState().isFirstRunOnly()) {
+      Map<String, ListenableFuture<Boolean>> activeTaskToResumeFutureMap = new HashMap<>();
```

Comment:
I am not sure if the term "active" task is used widely in the codebase. It seems ambiguous, especially considering that the next line mentions `activelyReadingTasks`. 

I would suggest simply using `tasksToResume`.

Reply:
It was supposed to mean a map from the set of activeTasks to their resumeFutures.
tasksToResume does seem simpler, I'll make the change

Reply:
Done